CHANGES_NEXT_RELEASE

```diff
@@ -7,6 +7,5 @@
 - Add: new value for URI param 'options': unique - like 'values', but avoiding duplicates (Issue #1050)
 - Add: check min length for IDs in v2 (Issue #1836)
 - Fix: PATCH /v2/entities correct interpretaion of null value and missing type in payload
-. Fix: PATCH /v2/entities missing value field (Issue #1789)
-
-
+- Fix: PATCH /v2/entities missing value field (Issue #1789)
+- Fix: parsing 'q-string-filter' in early stages and storing resulting objects for mongo-backend to access (q-part of Issue #1705)
```

Comment:
Not sure if this should be in the CNR... the change is transparant from the point of view of the user, without functinoal impact or fixing any existing problem.

Maybe "Hardening: XXX" should be a better entry category

(In addition, we typically use "mongoBackend" instead of "mongo-backend" in CNR entries)